services/movie_session.py

```diff
@@ -42,3 +43,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    seats = list(Ticket.objects.filter(movie_session_id=movie_session_id)
+                 .values("row", "seat"))
```

Comment:
Do not break line before `.`, it would be better to make line breaking after the parenthesis.